### DISCUSSION TITLE
Temperature sensor no decimal place

### DISCUSSION THREAD

**Discussion by chap470** [Rem0o/FanControl.Releases#1645] [General]
Why do some temperature sensors have decimal places and others do not? I have a water cooling and the water temperature sensor always goes only whole temperature steps up or down. Because I only move from 28°C - 33°C, the fan speed control is then very fast and step-like.
I have connected the water temperature sensor to my Asus B650E-F gaming motherboard.

**Comment by chap470** [comment]
No one an idea to fix this problem?

**Reply by Rem0o** [reply]
Not all temp sensors have decimal place resolution, that's just how it is.

**Reply by chap470** [reply]
That's strange. the sensor has a certain ohm resistance value. The mainboard or program converts this into a temperature. Do not understand how the sensor should be the problem.

**Reply by chap470** [reply]
The signal from the resistor is analog. Is a simple 10k Ohm resistor connected directly to my ASUS motherboard with a two wire cable.

**Reply by Rem0o** [reply]
What's the resolution of the ADC converting that value into a digital signal? Most likely 8 bit. So 256 values. Spread that over a 100 degree and you can't get decimal places. Just an educated guess. You can always look at the source code in LHM.
